.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Describe what you do "(>_<)"
+
+## Pics
+
+If you made changes screens, put here, please.
```

Comment:
"if you made chages on screens" or "if you changed some screen" wouldn't be better?

Reply:
Yep >_<